docs/source/defaulting_engine.rst

```diff
@@ -0,0 +1,69 @@
+.. _defaulting_engine:
+
+Defaulting Engine
+=================
+
+Radiant Labs understands that API users may not always know all of the details about the building being modeled. We have built a logical, statistics-based
+system to define default building characteristics. This defaulting engine uses tax assessor data, building permits, and NREL's ResStock building stock data.
+
```

Comment:
@kellyatradiantlabs naming ResStock is slightly too much information for my comfort level. Tax assess and permits is ok. Can you describe how ResStock got the data? Such as "regional building characteristics based on historical tax codes and x, y, z".